templates/slotViz.html

```diff
@@ -416,40 +403,46 @@
         if (!slotVizCountdown) {
           slotVizCountdown = document.createElement("div")
           slotVizCountdown.className = "mb-3"
-          $("#" + selector).prepend(slotVizCountdown)
+          $(selector).prepend(slotVizCountdown)
         }
         const slots = Math.ceil((hfTs - now) / 1000 / secondsPerSlot)
-        slotVizCountdown.innerHTML = `<b>Hardfork in:</b> ${getRelativeTime(luxon.DateTime.fromMillis(hfTs))} | ${slots} slots`
+        slotVizCountdown.innerHTML = `<b>${hardforkName} Hardfork in:</b> ${getRelativeTime(luxon.DateTime.fromMillis(hfTs))} | ${slots} slots`
```

Comment:
This adds an unnecessary space in front of "Hardfork" when no `${hardforkName}` is set.

Consider putting the space into `${hardforkName}` whenever a name is configured directly here:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/228994874181d60551b15675d1e8b8ca6a299efa/templates/slotViz.html#L11

Reply:
![image](https://user-images.githubusercontent.com/125363940/225248460-73eb7634-d742-4c48-a171-dea6a7170b13.png)